alacritty/src/renderer/mod.rs

```diff
@@ -1506,14 +1512,17 @@ impl Atlas {
             gl::PixelStorei(gl::UNPACK_ALIGNMENT, 1);
             gl::GenTextures(1, &mut id);
             gl::BindTexture(gl::TEXTURE_2D, id);
+            // We use an RGBA atlas that can handle colored glyphs.
+            // As they usually are outnumbered by regular text glyphs, this is slightly wasteful.
+            // Another way would be to have a separate atlas for RGBA and render in two passes.
```

Comment:
That was my main concern when I was adding RGBA information to solve this issue, so we should at least measure how it affects here to have RGBA instead of RGB, knowing that RGBA is very seldom thing. One more atlas seems like a good idea tbqh.

Reply:
So I want at least know if one more pass + atlas is better than RGBA for all the thing.

Reply:
One more `pro` for special atlas, is that colored glyphs require manual downsampling on Linux/BSD, so it saves perf if they will be live in there own atlas, however all of these require benchmarks and reports from GPU.

Reply:
Yes, I suppose benchmarking against master and a two-pass version is the next step.

Any pointer on what/how to benchmark would be appreciated. I suppose memory usage and some vtebench thing?

Reply:
Yes. `vtebench` has all information you need + render timer. But VRAM usage is likely the most interesting here. From what I've seen `vtebench` and `render_timer` were +- the same, however I'm not sure about memory and it's a bit harder to measure it.